### ISSUE TITLE
Adding new Class to Deeplabv3 Pre-trained Model 

### ISSUE THREAD

**Issue by essalahsouad** [tensorflow/models#6833]
### **System information**
**What is the top-level directory of the model you are using:**
https://github.com/tensorflow/models/tree/master/research/deeplab
**Have I written custom code (as opposed to using a stock example script provided in TensorFlow):**
No
**OS Platform and Distribution (e.g., Linux Ubuntu 16.04):**ubuntu 18.04
**TensorFlow installed from (source or binary):**binary
**TensorFlow version (use command below):**1.13.1 CPU
**Bazel version (if compiling from source):**N/A
**CUDA/cuDNN version:**N/A
**GPU model and memory:**N/A
### **Exact command to reproduce:**
run models/research/deeplab/deeplab_demo.ipynb
### Describe the problem
I tried train DeeplabV3 pre-trained model on a new class(eyes iris)
my dtat=RGB iris image+gray-scale iris label
as a result of the training step, a got those files:
graph.pbtxt
model.ckpt-1000.data-00000-of-00001
model.ckpt-1000.info
model.ckpt-1000.meta
that  i converted to frozen_inference_graph-1000.pb
in the test step a got this:
![Capture du 2019-05-21 11-45-04](https://user-images.githubusercontent.com/34586193/58089944-f3278680-7bbd-11e9-8886-f73ccb836f4f.png)
my model  does not recognize iris as a class
But i need to get something like this 
![iiiimgjcee](https://user-images.githubusercontent.com/34586193/58090259-bd36d200-7bbe-11e9-99c5-d443f085f59c.png)

### Source code / logs
[http://www.programmersought.com/article/2992281435/](url)
My dataset description looks like this:
![Capture du 2019-05-21 12-21-25](https://user-images.githubusercontent.com/34586193/58092045-0be66b00-7bc3-11e9-9c8b-7376dda308f1.png)

any help ??

**Comment by fabrizioschiano** [comment]
Hi, @essalahsouad , did you manage to do that, and will to share the process you adopted to do that?
